mongo_connector/doc_managers/mongo_doc_manager.py

```diff
@@ -60,10 +60,38 @@ def __init__(self, url, **kwargs):
             raise errors.ConnectionFailed("Failed to connect to MongoDB")
         self.namespace_set = kwargs.get("namespace_set")
         self.chunk_size = kwargs.get('chunk_size', constants.DEFAULT_MAX_BULK)
+        self.use_single_meta_collection = kwargs.get(
+                                        'use_single_meta_collection',
+                                        False)
+        self.meta_collection_name = kwargs.get(
+                                        'meta_collection_name',
+                                        constants.DEFAULT_META_COLLECTION_NAME)
+        self.meta_collection_cap_size = kwargs.get(
+                                    'meta_collection_cap_size',
+                                    constants.DEFAULT_META_COLLECTION_CAP_SIZE)
+        self.id_field = 'doc_id' if self.use_single_meta_collection else '_id'
```

Comment:
Why have a separate field called `doc_id` instead of putting this in `_id` as usual?